core/src/main/resources/org/apache/spark/ui/static/log-view.js

```diff
@@ -15,7 +15,9 @@
  * limitations under the License.
  */
 
-/* global $, getBaseURI */
+/* global $ */
+
+import {getBaseURI} from "./utils.js";
```

Comment:
It seems that we need to recover this. Could you review this, @yaooqinn ?
- https://github.com/apache/spark/pull/44896